tests/unit/test_characterProcessing.py

```diff
@@ -0,0 +1,21 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2020 NV Access Limited
+
+"""Unit tests for the characterProcessing module.
+"""
+
+import unittest
+from characterProcessing import SYMLVL_ALL
+from characterProcessing import processSpeechSymbols as process
+
+
+class TestComplex(unittest.TestCase):
+	"""Test the complex symbols rules.
+	"""
+
+	def test_group(self):

```

Comment:
This is a good start! Thanks for doing it. Apologies if you are already planning on expanding this, but I really want to also see tests for the `_replaceGroups` method. That's the part that contains the custom code. Please try to ensure each branch is hit with the tests.

Reply:
The test_group function was already triggering a call to `_replaceGroups`. But that wasn't exercising the '\\' case in group replacement so I have added a second test.